### PR TITLE
Release 2.9.7

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description

Looking for the PR for the upcoming v2.9.7 production release? Well you've just found it, congratulations! :tada:

This is based on v2.9.7-beta3 tag.

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] Verify that all feature flags are flipped appropriately
Notifications and pr quick view flagged to beta and dev.
image

- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated
- N/A

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+      "[Added] Support CLion as an external editor - #13739. Thanks @Pinzauti!",
+      "[Fixed] Don't show web flow committer on GitHub Enterprise Server - #13848",
+      "[Fixed] Cherry-pick success message always correctly reflects target branch - #13849",
+      "[Fixed] Fix error managing remotes under some circumstances - #13837",
+      "[Fixed] Add \"Copy Relative File Path\" option to changed files' context menu - #12589. Thanks @uttiya10!",
+      "[Fixed] Fix CLI tool on macOS Monterey - #13764. Thanks @spotlightishere!",
+      "[Fixed] Fix close button in full screen mode on macOS - #12838",
+      "[Fixed] Commit message dialog background styles match dialog - #13606",
+      "[Fixed] Ensure job steps on pull request check run list are always present - #13531",
+      "[Improved] Upgrade to Electron v16.0.8 - #13814",
```
Comment (sergiou87) [inline changelog.json]: AFAIK we don't announce these upgrades in prod release notes, right?

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+      "[Improved] Changes within lines are presented the same way in side-by-side and unified modes - #13838",
+      "[Improved] Select integration tab automatically when changing default editor  - #13856. Thanks @uttiya10!",
+      "[Improved] The check runs list for pull requests with multiple branches displays all actions workflow steps and headers - #13868",
+      "[Improved] Check run group headers and checks stay in view while scrolling the sub checks or job steps. - #13532",
+      "[Improved] Take alias into account when sorting repositories - #13429",
+      "[Improved] Support avatars on GitHub Enterprise Server - #13719",
+      "[Improved] Fetch before trying to follow a URL link to a specific branch - #13641. Thanks @Bestra!",
+      "[Improved] Add \"View on GitHub\" context menu option to repository list items - #13227. Thanks @lhvy!",
+      "[Improved] Signal when a commit summary is getting long - #2055. Thanks @Twixes!",
+      "[Improved] Remove unnecessary punctuation in appearance settings- #13715. Thanks @Pinzauti!"
```
Comment (sergiou87) [inline changelog.json]: Nit:
```suggestion
      "[Improved] Remove unnecessary punctuation in appearance settings - #13715. Thanks @Pinzauti!"
```

**Review by sergiou87 — CHANGES_REQUESTED** [review]
What a nice changelog!

File: changelog.json (hunk 1 above)
Comment (tidy-dev) [reply changelog.json]: Ah ok. I figured it was like when we have bumped Git, but yeah searching the releases we haven't in past. Thanks!

**Review by sergiou87 — DISMISSED** [review]
🚀

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+      "[Added] Support CLion as an external editor - #13739. Thanks @Pinzauti!",
+      "[Fixed] Don't show web flow committer on GitHub Enterprise Server - #13848",
+      "[Fixed] Cherry-pick success message always correctly reflects target branch - #13849",
+      "[Fixed] Fix error managing remotes under some circumstances - #13837",
+      "[Fixed] Add \"Copy Relative File Path\" option to changed files' context menu - #12589. Thanks @uttiya10!",
+      "[Fixed] Fix CLI tool on macOS Monterey - #13764. Thanks @spotlightishere!",
+      "[Fixed] Fix close button in full screen mode on macOS - #12838",
+      "[Fixed] Commit message dialog background styles match dialog - #13606",
+      "[Fixed] Ensure job steps on pull request check run list are always present - #13531",
+      "[Improved] Changes within lines are presented the same way in side-by-side and unified modes - #13838",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Changes within lines are presented the same way in side-by-side and unified diffs - #13838",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+      "[Improved] Select integration tab automatically when changing default editor  - #13856. Thanks @uttiya10!",
+      "[Improved] The check runs list for pull requests with multiple branches displays all actions workflow steps and headers - #13868",
+      "[Improved] Check run group headers and checks stay in view while scrolling the sub checks or job steps. - #13532",
+      "[Improved] Take alias into account when sorting repositories - #13429",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Take aliases into account when sorting repositories - #13429",
```

**Review by billygriffin — COMMENTED** [review]
Thanks! A couple minor suggestions that I'll go ahead and update but then good to go!

**Review by billygriffin — APPROVED** [review]
🚀 🤘

**Review by niik — APPROVED** [review]

**Pull request merged**